cli/src/template/ios/MoproApp/ContentView.swift

```diff
@@ -196,5 +203,102 @@ extension ContentView {
             print("\nUnexpected error: \(error)")
         }
     }
+
+    func runNoirProveAction() {
+        textViewText += "Generating zkEmail proof...\n"
+        
+        // Get the path to the SRS file in the app bundle
+        guard let srsPath = Bundle.main.path(forResource: "srs", ofType: "local") else {
+            textViewText += "Error: Could not find SRS file in app bundle\n"
+            return
+        }
+        
+        // Get the path to the input JSON file
+        guard let inputJsonPath = Bundle.main.path(forResource: "zkemail_input", ofType: "json") else {
+            textViewText += "Error: Could not find zkemail_input.json in app bundle\n"
+            return
+        }
+        
+        // Load and parse JSON
+        do {
+            let jsonData = try Data(contentsOf: URL(fileURLWithPath: inputJsonPath))
+            let decoder = JSONDecoder()
+            let inputData = try decoder.decode(ZkEmailInputTest.self, from: jsonData)
```

Comment:
No such `ZkEmailInputTest`

Reply:
fixed in [323ba30](https://github.com/zkmopro/mopro/pull/399/commits/323ba30633ccdf4ce6c601e06607de65b7ebb466)